packages/profile-sync-controller/src/controllers/user-storage/__fixtures__/mockStorage.ts

```diff
@@ -3,7 +3,8 @@ import encryption, { createSHA256Hash } from '../encryption';
 export const MOCK_STORAGE_KEY_SIGNATURE = 'mockStorageKey';
 export const MOCK_STORAGE_KEY = createSHA256Hash(MOCK_STORAGE_KEY_SIGNATURE);
 export const MOCK_STORAGE_DATA = JSON.stringify({ hello: 'world' });
-export const MOCK_ENCRYPTED_STORAGE_DATA = encryption.encryptString(
-  MOCK_STORAGE_DATA,
-  MOCK_STORAGE_KEY,
-);
+
+// NOTE - using encryption.encryptString directly in fixtures causes issues on mobile.
+// This is because this fixture is getting added in at run time. Will be improved once we support multiple exports
+export const MOCK_ENCRYPTED_STORAGE_DATA = () =>
+  encryption.encryptString(MOCK_STORAGE_DATA, MOCK_STORAGE_KEY);
```

Comment:
As mentioned, mobile has an issue with running this fixture.

This most likely is because our relative imports (barrel files), drag in these fixtures and need to be computed once pulled in.

We will explore multiple file exports, this should mostly alleviate this issue.

E.g.
```ts
// If want to reuse fixtures, we can have a separate endpoint for this. Likewise for other areas for this controller.
import {} from '@metamask/profile-sync-controller/fixtures'
```